exampleSite/config.yaml

```diff
@@ -8,7 +8,7 @@ title: Example Site
 disqusShortname: hugo-theme-stack
 
 # GA Tracking ID
-googleAnalytics:
+GoogleAnalytics:
```

Comment:
According to this line: https://github.com/gohugoio/hugo/pull/8287/files#diff-de73d0e34aa7f59e2e1e1ffc258cebde083c4f2111f9c7574b55e68dc438d237R38

It should be `googleAnalytics`, with initial lowercase.

Reply:
fix